Dockerfile.boringssl

```
@@ -23,17 +23,19 @@ RUN mkdir /install_root \
     --no-boot-update \
     && rm -rf /install_root/var/lib/swupd/*
 
-RUN swupd bundle-add os-core-dev c-basic python3-basic
+# GCC11 does not work with Envoy and we need to use GCC10.
+RUN swupd bundle-add os-core-dev c-extras-gcc10 python3-basic
+RUN ln -sf /usr/bin/gcc-10 /usr/bin/gcc && ln -sf /usr/bin/g++-10 /usr/bin/g++
```

We can keep `c-basic` and do it without symlinks like this:

```suggestion
RUN cd envoy && CC=gcc-10 CXX=g++-10 /root/.bazel/bin/bazel build --copt="-Wno-uninitialized" --copt="-Wno-maybe-uninitialized" --copt="-Wno-stringop-truncation" --verbose_failures -c opt -s //source/exe:envoy-static --extra_toolchains=@rules_python//python:autodetecting_toolchain_nonstrict \
```

---

In this case we do not need `c-basic` anymore. So we can replace it with `c-extras-gcc10`.
BTW, `os-core-dev` also contains `gcc11`.